src/devtools/mixins/entry-list.js

```diff
@@ -0,0 +1,12 @@
+import { scrollIntoView } from 'src/util'
+
+export default {
+  watch: {
+    inspectedIndex (value) {
+      this.$nextTick(() => {
+        const el = value === -1 ? this.$refs.baseEntry : this.$refs.entries[value]
+        el && scrollIntoView(document.querySelector('.left .scroll'), el, false)
```

Comment:
I recommend we use `.js-` prefixed classes for everything that is being handled by JS. It would allow us to more safely deal with the code and potential refactors.